docs/import/manage_disaster.js

```diff
@@ -61,48 +61,90 @@ let scoreBoundsMap;
 
 const scoreCoordinatesPath = Object.freeze([scoreCoordinatesAttribute]);
 
+const kickOffText = 'Kick off Data Processing (will take a while!)';
+const optionalWarningPrefix = '; warning: created asset will be missing ';
+
 /**
- * Checks that all fields that can be entered by the user have a non-empty
- * value. Does not check that assets actually exist, are of valid type, etc. If
- * all validation succeeds, enables kick-off button, otherwise disables and
- * changes button text to say what is missing.
+ * Checks that all mandatory fields that can be entered by the user have
+ * non-empty values. Does not check that assets actually exist, are of valid
+ * type, etc. If all validation succeeds, enables kick-off button, otherwise
+ * disables and changes button text to say what is missing.
+ *
+ * Some fields (Income, SVI) are optional. If they are missing, a separate
+ * message is displayed on the button, but it can still be enabled. If it is
+ * enabled, it is yellowed a bit to indicate the missing optional assets.
```

Comment:
So the color of the button for unable to kick off and able but missing is the same? What would you think about making them different? I think my slight preference is for anytime the button can actually be used for kicking off, it's green. But @ruthtalbot what do you think?

Reply:
It's supposed to be different: gray vs yellow vs green. There was a bug where if it was yellow, when it was disabled it stayed yellow, so thanks for pointing that out! Added an assertion in the test for that case.

@ruthtalbot seemed ok with yellow as opposed to just green (that's my preference as well) in the chat.

Reply:
Oh yeah, that sounds great too, as long as there's a difference between kicking off with all assets and kicking off with some optionals missing.